docs/index.md

```diff
@@ -193,21 +193,23 @@ The top line summarizes the context - we loaded the `set.mm` database
 (classical logic and ZFC set theory) and stopped reading the database before
 `2p2e4`.
 
-The next line is a tab bar, letting you select between *Settings* and
+The next line is the tab bar,
+letting you select between *Settings* and
 *Editor*.
 If you want to change the configuration of the tool, use *Settings*.
 Normally you''l be in the *Editor* tab (which lets you edit proofs),
 so we'll focus on that.
 
-Below the word "Editor" is an icon bar that shows different icons;
-each icon represents an actions you can do in the editor.
-We've already used one, the "+" symbol that represents adding a statement.
+Below the word "Editor" is the editor command icon bar.
+The editor command icon bar shows many different icons;
+each icon represents a command you can use to modify a proof.
+We've already used one, the "+" icon that represents adding a statement.
 The reference manual section
-[Editor command tab bar](#editor-command-tab-bar) discusses the
-icons and actions in more detail.
-You can hover over an icon to see what the command does.
+[Editor command icon bar](#editor-command-icon-bar) discusses each
+icon and the command each performs in more detail.
```

Comment:
Maybe a typo `and the command each performs` -> `and the command it performs`

Reply:
Carry on with long-click :-) !

It's a minor issue, but in general, it's best to consistently use *one* term for a given action. metamath-lamp sometimes says "substitution" and in other places says "replacement". The English words have essentially the meaning. However, the Metamath fundamental rule is already called "substitution", so I think it'd be better to use the word "replacement" for the icon so that we use different words for different actions. Anyway, that was my thinking :-).